sdk/src/main/java/com/hcaptcha/sdk/HCaptchaHtml.java

```diff
@@ -0,0 +1,209 @@
+package com.hcaptcha.sdk;
+
+final class HCaptchaHtml implements IHCaptchaHtmlProvider {
+
+    @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+    @Override
+    public String getHtml() {
+        return "<html>\n"
+                + "<head>\n"
+                + "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1, user-scalable=no\"/>\n"
```

Comment:
Would it be possible to keep the html in a `.html` file and package it in correctly at build time? Maybe via gradle tasks. This would ease the maintenance of the html and allow to benefit from IDE understanding the html code. Maybe as a follow up to not block current PR. @CAMOBAP

Reply:
Really good proposal, I have created https://github.com/hCaptcha/hcaptcha-android-sdk/issues/62